crates/swc_ecma_parser/src/lib.rs

```diff
@@ -187,6 +187,20 @@ impl Syntax {
         }
     }
 
+    pub fn disallow_assert_keywords(self) -> bool {
+        match self {
+            Syntax::Es(EsConfig {
+                disallow_assert_keywords,
+                ..
+            }) => disallow_assert_keywords,
+            #[cfg(feature = "typescript")]
+            Syntax::Typescript(TsConfig {
+                disallow_assert_keywords,
```

Comment:
Please remove this

Reply:
We should unconditionally follow the behavior of tsc

Reply:
Oh, I see. The issue I want to solve with this PR is that it prevents the `assert` keywords from being used in TypeScript, so I'll try to confirm if tsc can handle it.